### PR TITLE
Unit tests for utils folder

### PR DESCRIPTION
**Pull request by andre1050**
## What does this change?
Adds unit tests for `utils` folder.

## How can we measure success?
Test coverage for util functions used in the app.

### CODE REVIEW

File: ui/src/utils/testStatus.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { getTestStatus } from './testStatus';
+
+describe('getTestStatus', () => {
+  it(`should identity a running test`, () => {
```
Comment (tjmw) [inline ui/src/utils/testStatus.test.ts:4]: Small typo here?

```suggestion
  it(`should identify a running test`, () => {
```

File: ui/src/utils/truncate.test.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+describe('truncate', () => {
+  it(`should truncate a long string`, () => {
+    const text = 'Who looks outside, dreams; who looks inside, awakes.';
+    const obtainedText = truncate(text, 26);
+
+    expect(obtainedText).toBe('Who looks outside, dreams...');
+  });
+
+  it(`should truncate a long string`, () => {
```
Comment (tjmw) [inline ui/src/utils/truncate.test.ts:11]: Is this test description correct? Should it be something like `it('does not truncate a string shorter than the specified length')`?

File: ui/src/utils/testStatus.test.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    const obtainedStatus = getTestStatus(true);
```
Comment (tjmw) [inline ui/src/utils/testStatus.test.ts:5]: What do you think about assigning the `true` here to a `const` called `isEnabled` or similar? I think that might make it's intention clearer.

File: ui/src/utils/testStatus.test.ts (hunk 1 above)
Comment (andre1050) [reply ui/src/utils/testStatus.test.ts]: Yeah indeed, glad you spotted this.

File: ui/src/utils/truncate.test.ts (hunk 2 above)
Comment (andre1050) [reply ui/src/utils/truncate.test.ts]: Yeah exactly, bad copy & paste job there - I knew I needed tests for the tests :)

File: ui/src/utils/testStatus.test.ts (hunk 3 above)
Comment (andre1050) [reply ui/src/utils/testStatus.test.ts]: Good idea, thanks!

**Pull request merged**